### PR TITLE
Update contributing.md to have the right port value for Chrome.

### PR DESCRIPTION
**Pull request by wldcordeiro**
#596 mentioned this port change.

### CODE REVIEW

**Comment by clarkbw** [comment]
> this port change

Sorry, I should have clarified in the original issue.

The port you're looking for is in the Chrome command here: https://github.com/devtools-html/debugger.html/blob/master/CONTRIBUTING.md#chrome

localhost:8000 will show the debugger interface
But any mention of the examples directory, so http://localhost:8000/todomvc/ should use `7999` 
Also check for `/increment/` or `/pythagorean/`

**Comment by wldcordeiro** [comment]
@clarkbw I updated the PR I'm not seeing any mentions of `/increment/` or `/pythagorean/` that would need the replacement.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ To enabled Hot Reloading:
 
 ```json
 {
-  "hotReloading": true
```
Comment (wldcordeiro) [inline CONTRIBUTING.md]: Apparently my editor trimmed some whitespace automagically.

Comment (clarkbw) [inline CONTRIBUTING.md]: That's fine!

**Comment by jasonLaster** [comment]
Thanks for this fix!

**Pull request merged**